packages/location-management/src/components/LocationUnitGroupView/Table.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Table as AntTable, Menu, Dropdown, Button, Divider, Popconfirm } from 'antd';
+import { MoreOutlined } from '@ant-design/icons';
+import { LocationUnitGroup } from '../../ducks/location-unit-groups';
+import { getAccessToken } from '@onaio/session-reducer';
+import { useSelector } from 'react-redux';
+import { OpenSRPService } from '@opensrp/server-service';
+import { LOCATION_UNIT_GROUP_DELETE, URL_LOCATION_UNIT_GROUP_EDIT } from '../../constants';
+import { API_BASE_URL } from '../../configs/env';
```

Comment:
Let's retrieve the base URL from props

Reply:
@haseemVD have created a separate issue to deal with it .. it will be solved separately

Reply:
Created separate [issue](https://github.com/OpenSRP/web/issues/256) for this